tests/countries/test_cameroon.py

```
@@ -12,71 +12,136 @@
 
 from unittest import TestCase
 
-from holidays.countries.cameroon import Cameroon, CM, CMR
+from holidays.countries.cameroon import Cameroon
 from tests.common import CommonCountryTests
 
 
 class TestCameroon(CommonCountryTests, TestCase):
     @classmethod
     def setUpClass(cls):
```

Ruff (0.14.3)

Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)